### DISCUSSION TITLE
Using the Units Definitions Sources in other projects

### DISCUSSION THREAD

**Discussion by paulushub** [angularsen/UnitsNet#1432] [Q&A] [answered]
Hello,
Is there any license requirements or restrictions on using the [definitions sources](https://github.com/angularsen/UnitsNet/tree/master/Common/UnitDefinitions)?
I have ported the [JSR 385 - Units of Measurement API 2.2](https://github.com/unitsofmeasurement/unit-api) to .NET as part of an effort to port the Java/GeoAPI, which depends on the JSR 385. 
However, the reference implementation of the specification, [indriya](https://github.com/unitsofmeasurement/indriya), uses JavaCC to generate the parsers, which is not a readable code and I do not want to port that.  
I wish to use the definition sources to try an implementation of the .NET port. I will not use the code generations or related sources.
Please let know any license requirements or restriction - thank you.

**Comment by angularsen** [comment, marked as answer]
Everything is MIT-0 license, basically just do whatever 😄
